js/config.js

```diff
@@ -74,7 +74,9 @@ var _config = {
           STYLE_ATTRIBUTE_LIST: ['alignment-baseline', 'baseline-shift', 'clip', 'clip-path', 'clip-rule', 'color', 'color-interpolation', 'color-interpolation-filters', 'color-profile', 'color-rendering', 'cursor', 'direction', 'display', 'dominant-baseline', 'enable-background', 'fill', 'fill-opacity', 'fill-rule', 'filter', 'flood-color', 'flood-opacity', 'font-family', 'font-size', 'font-size-adjust', 'font-stretch', 'font-style', 'font-variant', 'font-weight', 'glyph-orientation-horizontal', 'glyph-orientation-vertical', 'image-rendering', 'kerning', 'letter-spacing', 'lighting-color', 'marker-end', 'marker-mid', 'marker-start', 'mask', 'opacity', 'overflow', 'pointer-events', 'shape-rendering', 'stop-color', 'stop-opacity', 'stroke', 'stroke-dasharray', 'stroke-dashoffset', 'stroke-linecap', 'stroke-linejoin', 'stroke-miterlimit', 'stroke-opacity', 'stroke-width', 'text-anchor', 'text-decoration', 'text-rendering', 'transform', 'transform-origin', 'unicode-bidi', 'vector-effect', 'visibility', 'word-spacing', 'writing-mode'],
           OUTPUT_SVG_STROKE_WIDTH: '1px',
           DISABLE_VECTOR_EFFECT: false,
-          DEFAULT_LINE_THICKNESS: '1px'
+          DEFAULT_LINE_THICKNESS: '1px',
+          DEFAULT_STROKE: 'red',
+          SVG_ATTRIBUTE_LIST: ["fill", "stroke", "stroke-width", "vector-effect", "cx", "cy", "r", "d", "x", "y", "height", "width"]
```

Comment:
What's the point of this?

Reply:
this was added so that i have a list of attributes that are being handled, and the order in which they should be present in the output. But since we have also have to add the `ignored-attributes`, I thought it would be better to just sort all the attributes in ascending order, and therefore ended up not using them. Should I remove `SVG_ATTRIBUTE_LIST`?

Reply:
Yes, you should remove any piece of code that you add along the way and never use.

Reply:
Done